run.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/bash
+set -x 
+set -e
+
+if [ -f .env ]; then
+  export $(echo $(cat .env | sed 's/#.*//g'| xargs) | envsubst)
+fi
+
+echo $BACKUP_CONTENT_FOLDER
+echo $GHOST_CONTENT_FOLDER
+echo $BLOG_NAME
+echo $GHOST_URL
+#check if the backup directory exists
+if [ -d "$BACKUP_CONTENT_FOLDER" ];
+then
+    rm -r $BACKUP_CONTENT_FOLDER
```

Comment:
Prefer not to automatically delete backups

`$(date +%d-%m-%y-time-%H-%M-%S)` may be useful for timestamping backups

Reply:
`tar -cvf $BLOG_NAME-backup-$(date +%d-%m-%y-time-%H-%M-%S) $GHOST_CONTENT_FOLDER`